src/Illuminate/Auth/SessionGuard.php

```diff
@@ -2,6 +2,13 @@
 
 namespace Illuminate\Auth;
 
+use Illuminate\Auth\Events\Attempting;
```

Comment:
These were intentionally not imported, surely?

Reply:
I can't see any reason why not.

Reply:
Because it was more readable with `Event\` in the code.